functions/pipeline/onboarding/onboarding-client.py

```diff
@@ -0,0 +1,40 @@
+import requests
+import json
+import pg8000
+
+# The following mock client imitates the CLI during the onboarding scenario for new images.
+# The expectation is that the CLI uploads images to a temporary blob store, then gets a list 
+# of URLs to those images and passes the list to an HTTP trigger function in the format of
+# a JSON string.  The HTTP trigger function creates rows in the database for the images,
+# retrieves the ImageId's for them, and then copies the images, each renamed as "ImageId.extension",
+# into a permanent blob storage container.  The HTTP function returns the list of URLs to
+# the images in permanent blob storage.
+
+print("\nTest client for CLI Onboarding scenario")
```

Comment:
Is this the start of the actual on-boarding client we will give CM? Is there a task for formalizing this? We can work on that together if you want to figure out what the right experience should be.

Reply:
No, this is a mock client that I'm using to test the function and verify that the client receives the correct responses after it runs